CovidTracking/CovidTracking/Scene/Ranking/RankingViewModel.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  RankingViewModel.swift
+//  CovidTracking
+//
+//  Created by Tiến on 5/20/21.
+//  Copyright © 2021 Tiến. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+import RxCocoa
+
+struct RankingViewModel: ViewModel {
+    
+    let mockData = [
+        DefaultLocation.Us,
+        DefaultLocation.Vietnam,
+        DefaultLocation.Brazin,
+        DefaultLocation.India,
+        DefaultLocation.Us,
+        DefaultLocation.Vietnam,
+        DefaultLocation.Brazin,
+        DefaultLocation.India,
+        DefaultLocation.Us,
+        DefaultLocation.Vietnam,
+        DefaultLocation.Brazin,
+        DefaultLocation.India,
+    ]
+    
+    var dataSource: BehaviorRelay<[Details]>
+    
+    init() {
+        dataSource = BehaviorRelay<[Details]>(value: mockData)
+    }
```

Comment:
```suggestion
    let dataSource = BehaviorRelay<[Details]>(value: mockData)
```

gộp luôn vào, khỏi cần init nữa